### PR TITLE
Implement browse year for both movies and shows

### PR DESCRIPTION
**Pull request by iRobsc**
fixes: #92

### CODE REVIEW

File: src/components/BrowsePage.js [hunk 1]
```diff
@@ -30,69 +36,62 @@ function BrowsePage({
...
-            render={() => (
-              <PosterGrid movies={movies} />
-            )}
-          />
-        </Switch>
-      </section>
+  } else if (isLoading) {
+    statusMsg = (
+      <div>
+        loading...
```
Comment (pajlilja) [inline src/components/BrowsePage.js]: Willy likes the loading text with capital L, else he will be annoyed about it for several weeks =(

Comment (iRobsc) [reply src/components/BrowsePage.js]: Ouch :( I can't hurt his feelings so I will have to change it.

Comment (willwull) [reply src/components/BrowsePage.js]: Lmao you guys

**Comment by iRobsc** [comment]
Yes we should merge this first so we can solve the merge conflicts in #95 instead.

**Comment by iRobsc** [comment]
Since we worked on the same thing at the same time there is a certainty that we would have merge conflicts.

File: src/containers/BrowseMoviesContainer.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import BrowsePage from "../components/BrowsePage";
-import { getMoviesFromType, getGenreMovies, getMovieGenres } from "../api/APIUtils";
+import { getMoviesFromType, getGenreMovies, getMovieGenres, getMoviesFromYear } from "../api/APIUtils";
+import createDebouncedFunc from "../utils/createDebouncedFunc";
 
 class BrowseMoviesContainer extends Component {
-  state = { movies: [], genreTitle: "", genres: [], isLoading: false, error: false };
+  state = { movies: [], genreTitle: "", genres: [], isLoading: false, error: "", searchWords: "" };
```
Comment (willwull) [inline src/containers/BrowseMoviesContainer.js]: Please put the state properties on their own line, this is really hard to read when there are so many values if they are on the same line

File: src/containers/BrowseMoviesContainer.js [hunk 3]
```diff
@@ -30,37 +46,52 @@ class BrowseMoviesContainer extends Component {
 
   getMoviesFromTab() {
     const { filter, id } = this.props.match.params;
-    this.setState({ movies: [], genreTitle: "", isLoading: true });
+    this.setState({ movies: [], genreTitle: "", isLoading: true, error: "" });
```
Comment (willwull) [inline src/containers/BrowseMoviesContainer.js]: Put the state properties on their own lines

File: src/containers/BrowseTvContainer.js [hunk 4]
```diff
@@ -1,10 +1,11 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import BrowsePage from "../components/BrowsePage";
-import { getShowsFromType, getGenreShows, getShowGenres } from "../api/APIUtils";
+import { getShowsFromType, getGenreShows, getShowGenres, getShowsFromYear } from "../api/APIUtils";
+import createDebouncedFunc from "../utils/createDebouncedFunc";
 
 class BrowseTvContainer extends Component {
-  state = { movies: [], genreTitle: "", genres: [], isLoading: false, error: false };
+  state = { movies: [], genreTitle: "", genres: [], isLoading: false, error: false, searchWords: "" };
```
Comment (willwull) [inline src/containers/BrowseTvContainer.js]: Put the state properties on their own lines

File: src/containers/BrowseTvContainer.js [hunk 5]
```diff
@@ -30,37 +46,52 @@ class BrowseTvContainer extends Component {
 
   getMoviesFromTab() {
     const { filter, id } = this.props.match.params;
-    this.setState({ movies: [], genreTitle: "", isLoading: true });
+    this.setState({ movies: [], genreTitle: "", isLoading: true, error: "" });
```
Comment (willwull) [inline src/containers/BrowseTvContainer.js]: Put the state properties on their own lines

**Comment by iRobsc** [comment]
Fixed @willwull

**Pull request merged**